.github/actions/process_test_results/action.yaml

```diff
@@ -21,7 +21,7 @@ runs:
       uses: mikepenz/action-junit-report@992d97d6eb2e5f3de985fbf9df6a04386874114d
       with:
         report_paths: results/*.xml
-        annotate_only: true
+        skip_annotations: true
```

Comment:
Did this end up significantly reducing total time for the publisher? Do we still want to replace with a different one?

Reply:
It did speed it up ~10x but that is still longer than the time it took to actually run the tests. I think we should try to find a replacement still.